packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -881,6 +884,112 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
     }
   }
 
+  @override
+  Future<bool> isSecondaryReset() async {
+    String? localPublicKey, remotePublicKey;
+    _logger.finer('Performing Remote Secondary reset check');
+    // Fetch EncryptionPublicKey from LocalSecondary
+    try {
+      localPublicKey = await getLocalSecondary()
+          ?.getEncryptionPublicKey(getCurrentAtSign()!);
+    } on Exception catch (e) {
+      _logger.severe(
+          'Exception caused fetch EncryptionPublicKey from LocalSecondary.'
+          ' Unable to complete reset check | Cause $e');
+      return false;
+    }
+    // Fetch EncryptionPublicKey from RemoteSecondary
+    try {
+      PLookupVerbBuilder plookup = PLookupVerbBuilder()
```

Comment:
what is the implication if we directly do an unauth lookup instead of auth plookup?